app/src/editor/canvas/components/ModuleSearch.pcss

```diff
@@ -202,3 +212,47 @@
 
 .dragHandle {
 }
+
+.dragElement {
```

Comment:
If this is a root element of the entire drag thing we could give it `font-size: 16px` and use
```
border-radius: 0.25em;
height: 3.75em;
max-width: 12.5em;
```

I'd like us to us them fancy units more often. :) I also defined `var(--um)` which is `16px`, pretty much. Can be used in CSS like so
```css
border-radius: calc(0.25 * var(--um));
```

This compiles into
```css
border-radius: 4px;
border-radius: calc(0.25 * var(--um));
```